R/linter_tags.R

```diff
@@ -0,0 +1,186 @@
+#' Get Linter metadata from a package
+#'
+#' Obtain a tagged list of all Linters available in a package.
+#'
+#' @param packages A character vector of packages to search for linters.
+#'
+#' @section Package Authors:
+#'
+#' To implement \code{available_linters()} for your package, include a file \code{inst/lintr/linters.csv} in your
+#' package.
+#' The CSV file must contain two columns, 'linter' and 'tags', and be UTF-8 encoded.
+#' Each row describes a linter by
+#'
+#' \enumerate{
+#' \item{its function name (e.g. \code{"assignment_linter"}) in the column 'linter'.}
+#' \item{space separated tags associated with the linter (e.g. \code{"style consistency default"}) in the column
+#'   'tags'.}
+#' }
+#'
+#' Tags should be snake_case.
```

Comment:
this doesn't really matter right?

Reply:
They must not contain spaces, but a "should" recommendation to stick to what we use seemed useful to me.

Reply:
to me "should" read as a requirement. maybe

> In accordance with the style guide, we recommend tags be in snake_case.

conveys it better?